docs/modules/about/pages/welcome.adoc

```diff
@@ -1,54 +1,40 @@
-= ALEC Documentation
 :imagesdir: ../assets/images
 
-== What is ALEC?
+= About ALEC
 
-The Architecture for Learning Enabled Correlation (ALEC) is a framework for logically grouping related faults (alarms) into higher level objects (situations) with OpenNMS.
+The Architecture for Learning Enabled Correlation (ALEC) provides a machine learning-powered framework for logically grouping related alarms into higher-level situations.
+As the correlation engines learn from users' network environments and their feedback, they will provide more accurate groupings and reduce the total number of notifications generated.
 
-== How does it work?
-
-=== Inventory in OpenNMS is converted to a graph
-
-Nodes, their components and their relations are used to build a graph.
-
-image::model.png[Model example,600]
+Because correlated alarms are likely to have been generated by the same underlying issue, alarm groupings provide a starting point for root cause analysis.
```

Comment:
```suggestion
Because correlated alarms are likely to have been generated by the same underlying issue, alarm groupings provide a starting point for root-cause analysis.
```